source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -58,10 +58,11 @@ const getPullRequestsAssociatedWithBranch = cache.function(async (): Promise<Rec
 	cacheKey: () => 'associatedBranchPullRequests:' + getRepo()!.nameWithOwner
 });
 
+// TODO: Replace this with `State--${prInfo.state.toLowerCase()}`
```

Comment:
If there’s no date it means it’s up for grabs immediately. Add a GHE mention with issue number of thats what you’re waiting for

Reply:
```suggestion
// TODO: Replace this with `State--${prInfo.state.toLowerCase()}` GHE #4202
```